sale_pricelist_triple_discount/test-requirements.txt

```
@@ -0,0 +1 @@
+odoo-addon-sale_triple_discount @ git+https://github.com/OCA/sale-workflow.git@refs/pull/3929/head#subdirectory=setup/sale_triple_discount
```

```suggestion
odoo-addon-sale_triple_discount @ git+https://github.com/OCA/sale-workflow.git@refs/pull/3929/head#subdirectory=sale_triple_discount
```
tests fail cause in 17.0 setup/ isn't necessary

---

done